### PR TITLE
Add icons for editor menus

### PR DESCRIPTION
**Pull request by akortunov**
Implements [#912](https://gitlab.com/OpenMW/openmw/issues/912).
Editor menus also take icons from UniversalId tables.

### CODE REVIEW

**Comment by Capostrophic** [comment]
Damn, this is what I never knew was sorely missing.

View submenu looks weird, but you're probably already aware of that.

**Comment by akortunov** [comment]
Ready for review and testing. @zinnschlag, can you take a look?

File: apps/opencs/model/world/universalid.cpp [hunk 1]
```diff
@@ -18,47 +18,60 @@ namespace
...
+        { CSMWorld::UniversalId::Class_RecordList, CSMWorld::UniversalId::Type_MagicEffects, "Magic Effects", ":./magic-effect.png" },
+        { CSMWorld::UniversalId::Class_RecordList, CSMWorld::UniversalId::Type_Lands, "Lands", ":./land-heightmap.png" },
+        { CSMWorld::UniversalId::Class_RecordList, CSMWorld::UniversalId::Type_LandTextures, "Land Textures", ":./land-texture.png" },
+        { CSMWorld::UniversalId::Class_RecordList, CSMWorld::UniversalId::Type_Pathgrids, "Pathgrids", ":./pathgrid.png" },
+        { CSMWorld::UniversalId::Class_RecordList, CSMWorld::UniversalId::Type_StartScripts, "Start Scripts", ":./start-script.png" },
+        { CSMWorld::UniversalId::Class_RecordList, CSMWorld::UniversalId::Type_MetaDatas, "Metadata", ":./metadata.png" },
+        { CSMWorld::UniversalId::Class_Transient, CSMWorld::UniversalId::Type_Redo, "Redo", ":./menu-redo.png" },
+        { CSMWorld::UniversalId::Class_Transient, CSMWorld::UniversalId::Type_Undo, "Undo", ":./menu-undo.png" },
+        { CSMWorld::UniversalId::Class_Transient, CSMWorld::UniversalId::Type_Preferences, "Preferences", ":./menu-preferences.png" },
```
Comment (zinnschlag) [inline apps/opencs/model/world/universalid.cpp]: This looks odd. UniversalIDs are used for stuff that either can have a subview or can be dragged around as icons, representing some part of  the content data. Menu items do not get an UniversalID.

Comment (akortunov) [reply apps/opencs/model/world/universalid.cpp]: Better now?

File: apps/opencs/view/doc/view.hpp [hunk 2]
```diff
@@ -66,6 +66,9 @@ namespace CSVDoc
 
             void closeEvent (QCloseEvent *event);
 
+            QAction* createMenuEntry(CSMWorld::UniversalId::Type type, QMenu* menu, const char* shortcutName);
```
Comment (zinnschlag) [inline apps/opencs/view/doc/view.hpp]: Good idea to factor out the menu item creation into a function. I should have done that in the first place.

**Pull request merged**